config.toml

```diff
@@ -0,0 +1,17 @@
+[config]
+"state_file" = "~/.busy_engineers_state.toml"
```

Comment:
FWIW, you don't actually need to quotes around the names. Removing them IMO makes for a more readable file. YMMV.

```toml
state_file = "~/.busy_engineers_state.toml"
```

Reply:
>Quoted keys follow the exact same rules as either basic strings or literal strings and allow you to use a much broader set of key names. Best practice is to use bare keys except when absolutely necessary.

You're right, I read this backwards. Will fix.